randovania/data/json_data/prime1.txt

```diff
@@ -2403,582 +2411,1266 @@ Ruins Entrance
 Asset id: 81183365
 > Dock to Dynamo Access; Heals? False
-  * Normal Door to Dynamo Access/Dock to Dynamo
+  * Missile Blast Shield to Dynamo Access/Dock to Dynamo
+  > Pickup (Missile Expansion)
+      Missile
+  > Pickup (Missile Expansion 2)
+      Morph Ball and Spider Ball
+
+> Pickup (Missile Expansion); Heals? False
+  * Pickup 25; Major Location? False
+  > Dock to Dynamo Access
+      Trivial
+
+> Pickup (Missile Expansion 2); Heals? False
+  * Pickup 26; Major Location? False
+  > Dock to Dynamo Access
+      Trivial
 
 ----------------
 Burn Dome Access
 Asset id: 4018058640
 > Dock to Burn Dome; Heals? False
   * Normal Door to Burn Dome/Dock to Burn Dome Access
+  > Dock to Energy Core
+      Morph Ball Bomb and Morph Ball
 
 > Dock to Energy Core; Heals? False
-  * Normal Door to Energy Core/Dock to Burn Dome Access
+  * Morph Ball Door to Energy Core/Dock to Burn Dome Access
+  > Dock to Burn Dome
+      Morph Ball
 
 ----------------
 West Furnace Access
 Asset id: 3262208600
 > Dock to Energy Core; Heals? False
   * Normal Door to Energy Core/Dock to West Furnace Access
+  > Dock to Furnace
+      Trivial
 
 > Dock to Furnace; Heals? False
   * Normal Door to Furnace/Dock to West Furnace Access
+  > Dock to Energy Core
+      Trivial
 
 ----------------
 Burn Dome
 Asset id: 1095301040
 > Dock to Burn Dome Access; Heals? False
   * Normal Door to Burn Dome Access/Dock to Burn Dome
+  > Pickup (Morph Ball Bombs)
+      Trivial
+  > Pickup (Missile Expansion)
+      Morph Ball Bomb and Morph Ball
+
+> Pickup (Morph Ball Bombs); Heals? False
+  * Pickup 28; Major Location? True
+  > Dock to Burn Dome Access
+      Trivial
+
+> Pickup (Missile Expansion); Heals? False
+  * Pickup 27; Major Location? False
+  > Dock to Burn Dome Access
+      Morph Ball Bomb and Morph Ball
 
 ----------------
 Furnace
 Asset id: 774997107
 > Dock to East Furnace Access; Heals? False
-  * Normal Door to East Furnace Access/Dock to Furnace
+  * Ice Door to East Furnace Access/Dock to Furnace
+  > Under Spider Track
+      Trivial
 
 > Dock to West Furnace Access; Heals? False
   * Normal Door to West Furnace Access/Dock to Furnace
+  > Pickup (Energy Tank)
+      Morph Ball Bomb and Morph Ball
+  > Under Spider Track
+      Morph Ball Bomb and Morph Ball and Spider Ball
 
 > Dock to Crossway Access West; Heals? False
-  * Normal Door to Crossway Access West/Dock to Furnace
+  * Morph Ball Door to Crossway Access West/Dock to Furnace
+  > Under Spider Track
+      Trivial
+
+> Pickup (Missile Expansion); Heals? False
+  * Pickup 29; Major Location? False
+  > Under Spider Track
+      Trivial
+
+> Pickup (Energy Tank); Heals? True
+  * Pickup 30; Major Location? False
+  > Dock to West Furnace Access
+      Morph Ball Bomb and Morph Ball
+
+> Under Spider Track; Heals? False
+  > Dock to East Furnace Access
+      Space Jump Boots
+  > Dock to West Furnace Access
+      Morph Ball Bomb and Morph Ball
+  > Dock to Crossway Access West
+      Trivial
```

Comment:
Oops I kinda forgot about this connection. Add morph to it

Reply:
Done